indexer-app/src/cmr/indexer/data/concepts/generic.clj

```diff
@@ -79,50 +79,48 @@
+        long-name (:LongName parsed-concept)
+        ;; We have already checked for approval in the ingest application.
         gen-name (util/safe-lowercase (get-in parsed-concept [:MetadataSpecification :Name]))
         gen-ver (get-in parsed-concept [:MetadataSpecification :Version])
-        approved (approved-generic? (keyword gen-name) gen-ver)]
-    (when approved
-      (let [index-data-file (format "schemas/%s/v%s/index.json" gen-name gen-ver)
-            index-file-raw (slurp (clojure.java.io/resource index-data-file))
-            index-data (json/parse-string index-file-raw true)
-            schema-keys [:LongName
-                         :Version
-                         :Description
-                         :RelatedURLs]
-            keyword-values (keyword-util/concept-keys->keyword-text
-                            parsed-concept schema-keys)
-            common-doc ;; fields common to all generic documents
-            {:concept-id concept-id 
-             :revision-id revision-id
-             :deleted deleted
-             :gen-name gen-name
-             :gen-name-lowercase (util/safe-lowercase gen-name)
-             :gen-version gen-ver
-             :generic-type (str gen-name " " gen-ver)
-             :provider-id provider-id
-             :provider-id-lowercase (util/safe-lowercase provider-id)
-             :keyword keyword-values
-             :user-id user-id
-             :revision-date revision-date
-             :native-id native-id 
-             :native-id-lowercase native-id 
-             :associations-gzip-b64 (associations->gzip-base64-str generic-assoc-ids)}
-             configs (gen-util/only-elastic-preferences (:Indexes index-data))
+        index-data-file (format "schemas/%s/v%s/index.json" gen-name gen-ver)
+        index-file-raw (slurp (io/resource index-data-file))
+        index-data (json/parse-string index-file-raw true)
+        schema-keys [:LongName
+                     :Version
+                     :Description
+                     :RelatedURLs]
+        keyword-values (keyword-util/concept-keys->keyword-text
+                        parsed-concept schema-keys)
+        common-doc ;; fields common to all generic documents
+        {:concept-id concept-id
+         :revision-id revision-id
+         :deleted deleted
+         :gen-name gen-name
+         :gen-name-lowercase (util/safe-lowercase gen-name)
+         :gen-version gen-ver
+         :generic-type (str gen-name " " gen-ver)
+         :provider-id provider-id
+         :provider-id-lowercase (util/safe-lowercase provider-id)
+         :keyword keyword-values
+         :user-id user-id
+         :revision-date revision-date
+         :native-id native-id
+         :native-id-lowercase native-id
+         :associations-gzip-b64 (associations->gzip-base64-str generic-assoc-ids)}
+        configs (gen-util/only-elastic-preferences (:Indexes index-data))
              ;; now add the configured indexes
```

Comment:
Really minor detail, but `;; now add the configured indexes` should be adjusted to the same indentation that `doc` is now at.

Reply:
Thank you!